src/components/Scenarios/ScenariosColumns.jsx

```diff
@@ -0,0 +1,83 @@
+import { inflictorWithValue } from 'components/Visualizations';
+import strings from '../../lang';
+import { transformations, formatSeconds } from '../../utility';
+
+const computeWinRate = row => (row.wins / row.games);
+
+const getTimeRange = (field, metadata) => {
+  let lower;
+  if (metadata.indexOf(field) !== 0) {
+    lower = metadata[metadata.indexOf(field) - 1];
+  } else {
+    lower = 0;
+  }
+  return `${formatSeconds(lower)} - ${formatSeconds(field)}`;
+};
+
+const getColumns = (f, metadata) => {
+  const columns =
+{
+  itemTimings: [{
+    displayName: strings.filter_hero_id,
+    field: 'hero_id',
+    sortFn: true,
+    displayFn: transformations.hero_id,
+  }, {
+    displayName: strings.scenarios_time,
+    field: 'time',
+    sortFn: row => row.time,
+    displayFn: (row, col, field) => getTimeRange(field, metadata.timings),
+  }, {
+    displayName: strings.scenarios_item,
+    field: 'item',
+    sortFn: true,
+    displayFn: (row, col, field) => inflictorWithValue(field),
+  }, {
+    displayName: strings.heading_win_rate,
+    field: 'games',
+    sortFn: computeWinRate,
+    percentBarsWithValue: row => row.games,
+    tooltip: strings.tooltip_winrate_samplesize,
```

Comment:
might want to abbreviate the sample size

Reply:
do you mean the word "sample size" or the number?